src/main/java/cubicchunks/asm/JvmNames.java

```diff
@@ -82,6 +83,7 @@ private JvmNames() {
         MATERIAL_WATER = MATERIAL + "WATER:" + MATERIAL,
         MUTABLE_BLOCK_POS_CONSTRUCT = MUTABLE_BLOCK_POS + "<init>(III)V",
         POOLED_MUTABLE_BLOCK_POS_SET_POS_D = POOLED_MUTABLE_BLOCK_POS + "setPos(DDD)" + POOLED_MUTABLE_BLOCK_POS,
+        TESSELLATOR_DRAW = TESSELLATOR + "draw()V",
```

Comment:
I didn't notice it before, but don't use JvmNames anymore, it's deprecated for a reason. It's from before Intellij + MinecraftDev plugin had autocomplete for this stuff.

Reply:
Alright, so should I just put the method names directly into the mixin annonations?

Reply:
yes. As MinecraftDev plugin for intellij has good autocomplete for it, there is no reason to use such constants for it. It just makes things harder to change, and harder to get help when you get something wrong